packages/react/src/isomorphicClerk.ts

```diff
@@ -882,6 +928,20 @@ export class IsomorphicClerk implements IsomorphicLoadedClerk {
     }
   };
 
+  mountWaitlist = (node: HTMLDivElement, props: WaitlistProps): void => {
+    if (this.clerkjs && this.#loaded) {
+      this.clerkjs.mountWaitlist(node, props);
+    } else {
+      this.premountWaitlistNodes.set(node, props);
+    }
+  };
+
+  unmountWaitlist = (node: HTMLDivElement): void => {
+    if (this.clerkjs && this.#loaded) {
+      this.clerkjs.unmountWaitlist(node);
+    }
+  };
```

Comment:
❓ Are we missing a `this.premountWaitlistNodes.delete(node);` here ?

Reply:
Good catch Pantelis 🙏